bundles/binding/org.openhab.binding.zwave/database/linear/wadwaz-1.xml

```diff
@@ -22,41 +22,8 @@
 			<id>0xFF</id>
 		</Class>
 	</CommandClasses>
-	<Configuration>
-		<Parameter>
-			<Index>1</Index>
-			<Type>short</Type>
-			<Default>0</Default>
-			<Size>2</Size>
-			<Label Lang="en">IN1 Alarm Cancellation Delay alarm Cancellation
-			</Label>
-			<Help Lang="en">Additional delay after an alarm from input IN1 has
-				ceased. The parameter allows you to specify additional time, after
-				which the input no. 1 alarm is cancelled once its violation has
-				ceased.
-			</Help>
-		</Parameter>
-		<Parameter>
-			<Index>2</Index>
-			<Type>byte</Type>
-			<Default>1</Default>
-			<Size>1</Size>
-			<Label Lang="en">Status change signaled by LEDs change signaled by
-			</Label>
-		</Parameter>
-	</Configuration>
 	<Associations>
 		<Group>
-			<Index>1</Index>
-			<Maximum>5</Maximum>
-			<Label lang="en">Input IN1</Label>
-		</Group>
-		<Group>
-			<Index>2</Index>
-			<Maximum>5</Maximum>
-			<Label lang="en">Tamper Button</Label>
-		</Group>
-		<Group>
 			<Index>3</Index>
```

Comment:
Thanks will do. Just added the GE Smart Switches I was mentioning
yesterday. About to pull for that commit.
On Jun 27, 2015 3:34 PM, "Chris Jackson" <notifications@github.com> wrote:

> In bundles/binding/org.openhab.binding.zwave/database/linear/wadwaz-1.xml
> <https://github.com/openhab/openhab/pull/2813#discussion_r33415926>:
>
> >  	<Associations>
> >  		<Group>
> > -			<Index>1</Index>
> > -			<Maximum>5</Maximum>
> > -			<Label lang="en">Input IN1</Label>
> > -		</Group>
> > -		<Group>
> > -			<Index>2</Index>
> > -			<Maximum>5</Maximum>
> > -			<Label lang="en">Tamper Button</Label>
> > -		</Group>
> > -		<Group>
> >  			<Index>3</Index>
>
> I’d recommend looking at SourceTree at some point. I used the git software
> for a while, but quickly found it lacking…
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab/pull/2813/files#r33415926>.
>